### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by imerdurgut**
Navbar login redirects to homepage, logout button redirects to homepage

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -22,13 +22,12 @@
...
               <%= link_to "Another action", "#", class: "dropdown-item" %>
-              <%= link_to "Log out", destroy_user_session_path, class: "dropdown-item", data: {turbo_method: :delete} %>
+              <%= button_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item", data: { turbo: false } %>
             </div>
-
           </li>
         <% else %>
           <li class="nav-item">
-            <%= link_to "Login", new_user_session_path, class: "nav-link" %>
+            <%= link_to "Login", new_user_session_path(from_navbar: true) %>
```
Comment (Lintong-source) [inline app/views/layouts/_navbar.html.erb:30]: I don't think we need this change. Or am I missing something?

Comment (imerdurgut) [reply app/views/layouts/_navbar.html.erb:30]: we can remove it

Comment (Lintong-source) [reply app/views/layouts/_navbar.html.erb]: nice. Done!

Comment (Lintong-source) [reply app/views/layouts/_navbar.html.erb]: we';; merge it

**Pull request merged**